{
  "_args": [
    [
      {
        "raw": "@google/cloud-errors@0.1.1",
        "scope": "@google",
        "escapedName": "@google%2fcloud-errors",
        "name": "@google/cloud-errors",
        "rawSpec": "0.1.1",
        "spec": "0.1.1",
        "type": "version"
      },
      "/Users/cfaws/AppDev/eliot-ai"
    ]
  ],
  "_from": "@google/cloud-errors@0.1.1",
  "_id": "@google/cloud-errors@0.1.1",
  "_inCache": true,
  "_location": "/@google/cloud-errors",
  "_nodeVersion": "7.3.0",
  "_npmOperationalInternal": {
    "host": "packages-12-west.internal.npmjs.com",
    "tmp": "tmp/cloud-errors-0.1.1.tgz_1482951324747_0.15319293388165534"
  },
  "_npmUser": {
    "name": "ofrobots",
    "email": "ofrobots@google.com"
  },
  "_npmVersion": "3.10.10",
  "_phantomChildren": {},
  "_requested": {
    "raw": "@google/cloud-errors@0.1.1",
    "scope": "@google",
    "escapedName": "@google%2fcloud-errors",
    "name": "@google/cloud-errors",
    "rawSpec": "0.1.1",
    "spec": "0.1.1",
    "type": "version"
  },
  "_requiredBy": [
    "/"
  ],
  "_resolved": "https://registry.npmjs.org/@google/cloud-errors/-/cloud-errors-0.1.1.tgz",
  "_shasum": "ebf6b70c0f03db6ed137d91f5bff1432de4bf83a",
  "_shrinkwrap": null,
  "_spec": "@google/cloud-errors@0.1.1",
  "_where": "/Users/cfaws/AppDev/eliot-ai",
  "author": {
    "name": "Google Inc."
  },
  "bugs": {
    "url": "https://github.com/googlecloudplatform/cloud-errors-nodejs/issues"
  },
  "contributors": [
    {
      "name": "Ali Sheikh"
    },
    {
      "name": "Cristian Cavalli"
    },
    {
      "name": "Matthew Loring"
    }
  ],
  "dependencies": {
    "@google/cloud-diagnostics-common": "0.3.0",
    "lodash": "^4.13.1"
  },
  "deprecated": "This module has been renamed to @google-cloud/error-reporting. Please use that module instead.",
  "description": "Node.js module for Google Stackdriver Error Reporting",
  "devDependencies": {
    "body-parser": "^1.15.1",
    "coveralls": "^2.11.11",
    "express": "^4.13.4",
    "hapi": "^13.4.1",
    "istanbul": "^0.4.3",
    "jsdoc": "git+https://github.com/jsdoc3/jsdoc.git",
    "jshint": "^2.9.2",
    "koa": "^1.2.0",
    "nock": "^9.0.0",
    "restify": "^4.1.0",
    "tape": "^4.5.1"
  },
  "directories": {},
  "dist": {
    "shasum": "ebf6b70c0f03db6ed137d91f5bff1432de4bf83a",
    "tarball": "https://registry.npmjs.org/@google/cloud-errors/-/cloud-errors-0.1.1.tgz"
  },
  "gitHead": "983e40185c71ffff2feaab26e5d76a57feddea89",
  "homepage": "https://github.com/googlecloudplatform/cloud-errors-nodejs#readme",
  "license": "Apache 2.0",
  "main": "index.js",
  "maintainers": [
    {
      "name": "google",
      "email": "github-admin@google.com"
    },
    {
      "name": "matthewloring",
      "email": "mattloring@google.com"
    },
    {
      "name": "ofrobots",
      "email": "ofrobots@google.com"
    }
  ],
  "name": "@google/cloud-errors",
  "optionalDependencies": {},
  "readme": "ERROR: No README data found!",
  "repository": {
    "type": "git",
    "url": "git+https://github.com/googlecloudplatform/cloud-errors-nodejs.git"
  },
  "scripts": {
    "coverage": "./bin/test.sh -l",
    "coveralls": "./bin/test.sh",
    "docs": "jsdoc -d docs index.js lib/",
    "integration-tests": "./node_modules/istanbul/lib/cli.js cover -x \"error-message.js\" ./node_modules/tape/bin/tape ./tests/integration/*.js",
    "style": "./node_modules/jshint/bin/jshint lib index.js",
    "test": "./node_modules/istanbul/lib/cli.js cover -x \"fuzzer.js\" ./node_modules/tape/bin/tape ./tests/unit/*.js"
  },
  "version": "0.1.1"
}
